kolibri/core/assets/src/styles/core-theme.styl

```diff
@@ -24,3 +24,6 @@ $core-bg-warning = #fff3e1
 
 $core-text-error = #b93329
 $core-bg-error = #fee9e7
+
+$core-z-index-top = 100
+$core-z-index-bottom = 0
```

Comment:
you'll want to coordinate with @christianmemije on z-index updates - I know he's working on cleaning up our z-index mess. See e.g. https://github.com/learningequality/kolibri/pull/1082

Having some gobally defined constants seems like a good idea - but they should be in _definitions.styl_, not _core-theme.styl_. (Things in _core-theme_ are potentially overridable, whereas _definitions_ will remain the same across all themes.)